src/components/v5/shared/CardWithSections/CardWithSections.tsx

```diff
@@ -11,7 +11,7 @@ const CardWithSections: React.FC<CardWithSectionsProps> = ({
   footerClassName,
 }) =>
   sections.length ? (
-    <Card className="w-full" withPadding={false}>
+    <Card className="w-full overflow-hidden" withPadding={false}>
```

Comment:
What is this overflow added here for?

Reply:
There was a bug with borders
![image](https://github.com/JoinColony/colonyCDapp/assets/76940796/75fe2ab6-8dfb-48aa-8ef1-003137b1ec10)